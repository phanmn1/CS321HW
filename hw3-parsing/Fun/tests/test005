int global = foo();

int foo(int num, boolean hah)
{
   ;
   print test(blah, blah, blah);

   return foo(int num);
// shouldn't be able to pass an 'int' anything as the expression to a function call.
}
