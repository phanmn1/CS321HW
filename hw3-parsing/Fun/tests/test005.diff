1c1
< Invalid syntax at (14,8), (
---
> Invalid syntax at (17,1), (
