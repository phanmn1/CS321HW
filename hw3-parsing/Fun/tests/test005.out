Invalid syntax at (17,1), (
